optaplanner-docs/src/main/asciidoc/ConstraintStreams/ConstraintStreams-chapter.adoc

```diff
@@ -230,11 +450,14 @@ The following example finds a computer which runs the most power-demanding proce
 == Variant implementation types
 
 Constraint streams come in two flavours, a default implementation using Drools under the hood and a pure Java-based
-implementation called _Bavet_. Drools-based implementation is more feature-complete.
+implementation called _Bavet_.
+Drools-based implementation is more feature-complete.
+Both of these implementations use the same `ConstraintProvider` API, no Java code changes are necessary to switch
+between the two.
 
-Bavet is an experimental implementation focusing on raw speed, which often significantly outperforms the Drools
+Bavet is an experimental implementation focusing on raw speed, which often significantly outperforms the default
 implementation, or even Drools itself.
```

Comment:
"or even Drools itself". That's weird. What we mean is "scoreDRL". But I wouldn't mention that at all here - that's an impl detail. I'd remove "or even Drools itself".